package.json

```diff
@@ -32,6 +32,7 @@
     "dev:client": "grunt clean:client && webpack-dev-server --config=./webpack/webpack.develop.js",
     "dev:client:prod": "grunt clean:client && webpack-dev-server --config=./webpack/webpack.production.js",
     "dev:nodl": "grunt clean:server && grunt clean:client && grunt build:server && grunt start:client",
+    "dev:extension": "grunt build && grunt install:extension",
```

Comment:
might be easier to use `make local -f build/docker-extension/Makefile` here instead of burying this single line in layers of grunt

Reply:
thanks, resolved